boards/shields/wnc_m14a2a/Kconfig.shield

```diff
@@ -0,0 +1,6 @@
+# Copyright (c) 2019 Linaro Limited
+# SPDX-License-Identifier: Apache-2.0
+
+config SHIELD_WNC_M14A2A
+	bool
+	default $(shields_list_contains,wnc_m14a2a)
```

Comment:
Could also use `def_bool` here. It's pretty common for non-configurable "helper" symbols like this one.

```
config SHIELD_WNC_M14A2A
	def_bool $(shields_list_contains,wnc_m14a2a)
```

Reply:
Done, txs!